.github/workflows/ci.yml

```
@@ -644,3 +644,34 @@ jobs:
         run: |
           meson --backend=ninja build --cross-file test/arm64cl.txt
           meson test -C build --print-errorlogs $(meson test -C build --list | grep -v emul)
+
+  linux-gcc-loongarch64:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v3
+    - name: CPU Information
+      run: cat /proc/cpuinfo
+    - name: Install APT Dependencies
+      run: sudo apt-get install -y ninja-build ninja-build meson
+    - name: loongarch64-toolchain
+      run: |
+        wget https://github.com/loongson/build-tools/releases/download/2022.09.06/loongarch64-clfs-7.3-cross-tools-gcc-glibc.tar.xz
+        tar -xvf loongarch64-clfs-7.3-cross-tools-gcc-glibc.tar.xz -C /opt
+        wget https://github.com/loongson/build-tools/releases/download/2022.09.06/qemu-loongarch64
+        chmod +x qemu-loongarch64
```

https://github.com/loongson/build-tools#readme states that the sourcecode for this is https://download.qemu.org/qemu-8.0.0.tar.xz ; would it work to install http://deb.debian.org/debian/pool/main/q/qemu/qemu-user-static_8.0+dfsg-4_amd64.deb ?
```suggestion
        wget http://deb.debian.org/debian/pool/main/q/qemu/qemu-user-static_8.0+dfsg-4_amd64.deb
        apt install ./qemu-user-static_8.0+dfsg-4_amd64.deb
```